lia-core/src/main/java/com/lithium/community/android/model/helpers/LiImage.java

```diff
@@ -85,36 +67,25 @@ public Long getWidth() {
         return width;
     }
 
-    public void setWidth(LiBaseModelImpl.LiInteger result) {
-        this.width = result.getValue();
+    public void setWidth(Long result) {
```

Comment:
I would suggest we stick to primitive long or Long.. auto boxing is expensive

Reply:
agree.

Reply:
There are too many such all over the code, I think we can live with it.

Reply:
LiInt and LiLong were earlier using Long.. I guess we can stick to that.. no point introducing primitives now

Reply:
sure